google/cloud/security/common/data_access/group_dao.py

```diff
@@ -25,28 +25,68 @@
 class GroupDao(dao.Dao):
     """Data access object (DAO) for Groups."""
 
-    def get_group_users(self, resource_name, timestamp):
-        """Get the group members who are users.
+    def get_group_id(self, resource_name, email, timestamp):
+        """Get the id of a group from an email.
+
+        Args:
+            resource_name: String of the resource name.
+            group_id: String of the group id.
+            timestamp: The timestamp of the snapshot.
+
+        Returns:
+             String of the group id.
+        """
+        sql = select_data.GROUP_ID.format(email, timestamp)
+        result = self.execute_sql_with_fetch(resource_name, sql, None)
```

Comment:
nit: passing in none here is confusing and I've seen this a couple times. Can we just give execute_sql_with_fetch a default for that kwarg.